rgw/v2/tests/s3_swift/test_swift_basic_ops.py

```diff
@@ -585,7 +590,7 @@ def test_exec(config, ssh_con):
                 log.info(f"Delete container {container_name} with container owner")
                 rgw.delete_container(container_name)
 
-        if config.test_ops.get("enable_version_by_s3", False):
+        elif config.test_ops.get("enable_version_by_s3", False):
```

Comment:
@ckulal , can we retain using if statement, as we might endup executing the 'else' section for every run.

Reply:
done